deployment/performance-tuning-single-process.md

```diff
@@ -76,6 +76,11 @@ Here's a quote from the documentation:
 
 So, the default GC behavior does not call full GC until the number of old objects reaches `2.0 * before old objects`. This improves the throughput but it grows the total memory usage. This setting is not good for the low resource environment e.g. a small container. For such cases, try `RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR=0.9` or `RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR=1.2`.
 
+{% hint style='danger' %}
+There is a case that `RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR=0.9` causes a delay of launching Fluentd.
+As excessively tune to reduce memory usage causes performance degrade, configure `RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR` parameter by your own risk.
+{% endhint %}
+
```

Comment:
How about more aggressively preventing the use of values less than 1.0?

```suggestion
So, the default GC behavior does not call full GC until the number of old objects reaches `2.0 * before old objects`. This improves the throughput but it grows the total memory usage. This setting is not good for the low resource environment e.g. a small container. For such cases, try `RUBY_GC_HEAP_OLDOBJECT_LIMIT_FACTOR=1.2`, etc.

{% hint style='danger' %}
Do not set a value lower than `1.0` unless there is a special reason, because there are cases where setting a value lower than `1.0` causes a delay of launching Fluentd.
{% endhint %}
```

Reply:
Thanks, a bit modified it.